viewing_party/party.py

```diff
@@ -1,23 +1,147 @@
 # ------------- WAVE 1 --------------------
+from tests.test_constants import MOVIE_TITLE_1, RATING_1
 
+#make a dictioary with three parameters if three of them are truthy
+#if any of them are none function returns none
 def create_movie(title, genre, rating):
-    pass
+    
 
-# -----------------------------------------
+    new_movie=dict([("title",title),("genre", genre),("rating",rating)])
+  
+    
+    if title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None

```

Comment:
Let's combine these if statements into two lines, so we aren't repeating code over and over! After that, we should put the if statement (sometimes called a "guard clause") at the top of the function. 

That way the function will check the parameters first, then get out of there if they aren't correct. Otherwise we end up using memory space on a dictionary first instead of checking if the parameters are correct.

```suggestion
    if not title or not genre or not rating:
        return None
        
    new_movie = {
        "title": title,
        "genre": genre,
        "rating": rating
        }
```